src/index.js

```diff
@@ -0,0 +1,132 @@
+}
+
+const changeTempFontColor = () => {
+  if (state.temp <= 49) {
+    color = "#3F50B5";
+    landscape = "🤧⛄️🌲⛄️🛷🌲⛷🗻🌲⛸🏂⛄️🍂❄️";
+  } else if (state.temp >= 50 && state.temp<= 59) {
+    color = "#00BCD4";
+    landscape = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+  } else if (state.temp >= 60 && state.temp  <= 69) {
+    color = "#FEC106";
+    landscape = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+  } else if (state.temp >= 70 && state.temp  <= 79) {
+    color = "#FF9802";
+    landscape = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+  } else {
+    color = "#F44336";
+    landscape = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+  }
+  document.getElementById("temperature").style.color = color;
+  document.getElementById("landscapeContainer").textContent = landscape;
+}
+
+const submitInput = () => {
+  
+  const subheaderText = document.querySelector("#inputFormContents");
+  const inputButton = document.querySelector("#inputButton");
+  
+  if (isReset) {
+    subheaderText.textContent = "Input your name and location for a weather report made just for you!";
+    document.getElementById("fname").value = "";
+    document.getElementById("location").value = "";
+    inputButton.textContent = "Submit"
+    isReset = false;
+  } else {
+    let guestName = document.getElementById("fname").value;
+    let location = document.getElementById("location").value;
+    subheaderText.textContent = `Welcome, ${guestName} :) Here's the weather for lovely ${location}!`;
+    inputButton.textContent = "Reset";
+    isReset = true;
+  }
+}
+
+const changeSky = (currentSelection) => {
+  let emojis = ''
+
+  if (currentSelection == "sunny") {
+    emojis = "☀️ 😎 ☀️";
+  } else if (currentSelection == "cloudy") {
+    emojis = "☁️ ☁️ ☁️";
+  } else if (currentSelection == "rainy") {
+    emojis = "🌧 💧 🌦";
+  } else if (currentSelection == "snowy") {
+    emojis = "🌨 ❄️ 🌨";
+  } else {
+    emojis = "🐻";
+  }
+
+  document.querySelector("#h1Header").textContent = `${emojis} MyWeather ${emojis}`;
+}
+
+const registerEventHandlers = (event) => {
+  const tempButtonUp = document.querySelector("#plusButton");
+  tempButtonUp.addEventListener("click", function(){changeTemp('up')});
+
+  const tempButtonDown = document.querySelector("#minusButton");
+  tempButtonDown.addEventListener("click", function(){changeTemp('down')});
+
+  const celsiusSlider = document.querySelector("#switch");
+  celsiusSlider.addEventListener("change",function(){
+    if (isFahrenheit) {
+      isFahrenheit = false;
+    } else {
+      isFahrenheit = true;
+    }
+    changeTemp("");
+  })
+
+  const vibeChangeButton = document.querySelector("#footerMsg");
+  vibeChangeButton.addEventListener("click", swapStyleSheet);
+
+  const submitInputButton = document.querySelector("#inputButton");
+  submitInputButton.addEventListener("click", submitInput);
+
+  const skySelectDropdown = document.querySelector("#sky");
+  skySelectDropdown.addEventListener("change", function(){
+    let currentSelection = skySelectDropdown.value
+    changeSky(currentSelection)
+  })
```

Comment:
I'm not really bothered by explicitly coding this function to use `skySelectDropdown` (since it's a one-off anonymous method). But remember that event handlers are passed information about the event which we could use instead, which can be useful if a function might need to be registered on multiple UI elements.

```js
  skySelectDropdown.addEventListener("change", function(event){
    let currentSelection = event.target.value
    changeSky(currentSelection)
  })
```